ga4gh/client.py

```diff
@@ -121,14 +126,15 @@ def _doRequest(self, httpMethod, url, protocolResponseClass,
         Performs a request to the server and returns the response
         """
         headers = {}
-        params = self._getAuth()
-        params.update(httpParams)
+        params = httpParams
```

Comment:
We should make a copy of httpParams, since modifying the parameter could have subtle side effects.